components/gitpod-protocol/src/protocol.ts

```diff
@@ -1205,6 +1208,27 @@ export interface Project {
     deleted?: boolean;
 }
 
+export interface ProjectInfo {
```

Comment:
Why copying the whole `Project` shape instead of referencing it?

Reply:
Just learned that `DBProject` isn't implementing `Project` at all.
So, yeah it doesn't make sense to duplicate. 

Besides that, we introduced `xInfo` shapes to augment entities, to make it works nicer with the frontend.